src/test/reducer.test.js

```diff
@@ -0,0 +1,74 @@
+const {ADD, DELETE, UPDATE_COUNT, START, END} = require("../constants");
+const {reducer} = require("../reducers/reducer");
+
+test('pass addImages action', () => {
+    const state = {
+        images: ["first", "second"]
+    };
+    const action = {
+        type: ADD,
+        data: ["third", "fourth"]
+    }
+    expect(reducer(state, action)).toStrictEqual({...state, images: [...state.images, ...action.data]});
```

Comment:
I've never used `toStrictEqual`. Could you please explain why you are using `toStrictEqual` rather than `toEqual`?